wiring/inc/spark_wiring_system.h

```diff
@@ -261,17 +261,21 @@ class SystemClass {
 		set_flag(flag, false);
     }
 
+    inline bool enabled(system_flag_t flag) const {
+        return get_flag(flag) != 0;
+    }
+
 
 private:
 
-    inline uint8_t get_flag(system_flag_t flag)
+    static inline uint8_t get_flag(system_flag_t flag)
     {
         uint8_t value = 0;
         system_get_flag(flag, &value, nullptr);
         return value;
     }
 
-    inline void set_flag(system_flag_t flag, uint8_t value)
+    static inline void set_flag(system_flag_t flag, uint8_t value)
```

Comment:
What's the reason behind making this static? I'd like to avoid someone writing SystemClass::set_flag() since we may in future need to maintain state, requiring an instance method, i.e. `System.set_flag()`. 

In practice, the code produced will be the same since the compiler will optimize out `this` when inlining the body.